### PR TITLE
DOCSP-37839 Split mongorestore into multiple topics

### PR DESCRIPTION
**Pull request by nvillahermosa-mdb**
## DESCRIPTION
- Reduces the size of the mongorestore reference topic by splitting off the install info, behavior, and examples into respective subtopics
- Removes unused extract

## STAGING
[mongorestore](https://preview-mongodbnvillahermosamdb.gatsbyjs.io/database-tools/DOCSP-37839-split-mongorestore-page/mongorestore/)
[mongorestore Compatibility and Installation](https://preview-mongodbnvillahermosamdb.gatsbyjs.io/database-tools/DOCSP-37839-split-mongorestore-page/mongorestore/mongorestore-compatibility-and-installation/)
[mongorestore Behavior, Access, and Usage](https://preview-mongodbnvillahermosamdb.gatsbyjs.io/database-tools/DOCSP-37839-split-mongorestore-page/mongorestore/mongorestore-behavior-access-usage/)
[mongorestore Examples](https://preview-mongodbnvillahermosamdb.gatsbyjs.io/database-tools/DOCSP-37839-split-mongorestore-page/mongorestore/mongorestore-examples/)

## JIRA
https://jira.mongodb.org/browse/DOCSP-37839

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=660eedf23a5920a29121b569

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by kennethdyer — APPROVED** [review]
Thanks @nvillahermosa-mdb, LGTM

**Comment by nvillahermosa-mdb** [comment]
@sarahemlin Can you take a look at this PR for breaking up the mongorestore page? Thanks!

**Comment by sarahemlin** [comment]
@sarah-olson-mongodb, I know they were left in on the first two tools, but could we not use the Learn More sections at the bottom?  With only three pages and the previous/next links at the bottom, too, this feels redundant to me.

File: source/mongorestore/mongorestore-behavior-access-usage.txt [hunk 1]
```diff
@@ -0,0 +1,178 @@
+.. _mongorestore-behavior-access-usage:
+
+============================================
+mongorestore Behavior, Access, and Usage
```
Comment (sarahemlin) [inline source/mongorestore/mongorestore-behavior-access-usage.txt:4]: @nvillahermosa-mdb on the other pages we've split, this page was just <tool-name> Behavior, even though Access & Usage are in the page hierarchy.  I'd vote for keeping it to Behavior for simplicity's sake if you're good with that.

**Review by sarahemlin — APPROVED** [review]

**Pull request merged**